examples/CharacterText/accessibility.ts

```diff
@@ -1,3 +1,4 @@
+import * as txt from "txt";
```

Comment:
So glad I got this working. In the end it was the combination of setting `compilerOptions.paths` in `tsconfig.json`, and also excluding modules using the `--global` flag when calling rollup.